build.gradle

```diff
@@ -55,202 +21,21 @@ dependencies {
     )
 }
 
-asciidoctorj {
- version = '1.5.4'
-}
-
 
+//all available docToolchain modules
+//the first is mandatory
+apply from: 'scripts/AsciiDocBasics.gradle'
```

Comment:
I would split out all generateX tasks to separate .gradle files as well, so that if a project doesn't need e.g. PDF then it's not bloated with unnecessary stuff.

Reply:
I thought about this too, but it seems that it's not easy to split the AsciiDoc tasks because of the 
task.withType(asciidoctor)
statement. I even hat to go back to the old plugin mechanism in order to get it this far...